src/main/java/com/SAS/crudoperations/TransactionCRUD.java

```diff
@@ -1,39 +1,70 @@
 package com.SAS.crudoperations;
 
+import com.SAS.User.TeamOwner;
 import com.SAS.transaction.Transaction;
+import com.SAS.transaction.TransactionType;
 import org.springframework.jdbc.core.JdbcTemplate;
 
+import javax.naming.LimitExceededException;
+import java.util.LinkedList;
+import java.util.List;
+
+/**
+ * The class represents all the transactions operations with the db
+ */
 public class TransactionCRUD {
-    private JdbcTemplate jdbcTemplate;
+    private static JdbcTemplate jdbcTemplate;
 
     public void setTemplate(JdbcTemplate jdbcTemplate) {
         this.jdbcTemplate = jdbcTemplate;
     }
 
-    public int saveTransaction(Transaction newTransaction){
-
-        //TEST
-        String team_name = "macabi";
-        String type = "INCOME";
-        String date = "2020-03-30";
-        String description = "bought arnon";
-        int teamOwnerId = 5;
-        double amount = 1000.0;
-        int id = 9;
+    /**
+     * The function receives transaction and add it to the db
+     * @param newTransaction
+     * @return
+     */
+    public static int addTransactionToDB(Transaction newTransaction){
+        String team_name = newTransaction.getTeam().getName();
```

Comment:
It should only get the parameters from the teamManagement - as String transactionName, type, and so on.